packages/desktopjs/src/registry.ts

```diff
@@ -68,16 +68,18 @@ export function resolveContainer(param1?: boolean | any, param2?: any): Containe
     if (!force && container) {
         return container;
     }
-
-    for (const entry in registeredContainers) {
-        try {
-            if (registeredContainers[entry].condition(options)) {
-                return container = registeredContainers[entry].create(options);
-            }
-        } catch (e) {
-            console.error("Error resolving container '" + entry + "': " + e.toString());
+    let registration: ContainerRegistration;
+    try {
+        for (let i: number =0; i< registeredContainers.length; i++) { // tslint:disable-line
+            const testReg: ContainerRegistration = registeredContainers[i].registration;
+            registration = testReg.condition(options) ? testReg : registration;
         }
+        container = registration.create(options);
+    } catch (e) {
+        console.error(`Error resolving container : ${e.toString()}`);
```

Comment:
This lost the name of the container which failed to resolve.  Can you add the registry name back in?